src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Infrastructure/TestConstants.cs

```diff
@@ -12,9 +12,13 @@ static class TestConstants
         
         // General 
         internal const string SessionPrefix = "session";
-        internal const int MaxAttemptsCount = 5;
+        internal const int MaxAttemptsCount = 5;        
```

Comment:
nit: whitespace.

Reply:
Thanks.    Worth calling out also, that this `MaxAttemptsCount` is not the "please retry a failed test this many times" control.   This was an existing value used in some tests to control the number of times that it attempts to receive messages that were sent and similar operations.

The `LiveTestDefaultMaxAttempts ` a few lines below is the test failure retry control.   It's currently set to just a single retry on failures to try and avoid hiding legitimate issues.